### DISCUSSION TITLE
`搜索推荐` 插件和 `自定义顶栏` 组件是不能一起使用的吗？

### DISCUSSION THREAD

**Discussion by SekiBetu** [the1812/Bilibili-Evolved#2405] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/38283893/138024269-9b917de1-cbe2-4d08-87e2-702c3ce375ca.png)

![image](https://user-images.githubusercontent.com/38283893/138024280-c07b8ac9-fef5-420a-855d-fb4038c6d9bd.png)

`搜索推荐` 插件和 `自定义顶栏` 组件是不能一起使用的吗？

一起使用的话 `搜索推荐` 插件无法应用到 `自定义顶栏` 组件？

![image](https://user-images.githubusercontent.com/38283893/138024388-796b61e3-85f3-4e63-b159-b6ba7565d905.png)

关掉 `自定义顶栏` 组件后 `搜索推荐` 插件生效
![image](https://user-images.githubusercontent.com/38283893/138024404-ccecf21d-3f64-41c3-a461-6914a8dbd0a8.png)

**Comment by the1812** [comment, marked as answer]
推荐的是搜索框里那个词, 不影响下面的列表, 如果没有这个插件, 搜索框里就只显示 `搜索`
